run_service.sh

```diff
@@ -1272,7 +1276,7 @@ export STOP_TRADING_IF_STAKING_KPI_MET=true
 export RESET_PAUSE_DURATION=45
 export MECH_WRAPPED_NATIVE_TOKEN_ADDRESS=$WXDAI_ADDRESS
 export MECH_CHAIN_ID=ethereum
-export TOOLS_ACCURACY_HASH=QmXhah4pLENsc4pbDTdKmw22GLGb4Z1FSG3FiEsF9ayxxd
+export TOOLS_ACCURACY_HASH=QmYna61uWDGELf4kTEKSfjQaKDUEv9ALSTce6CGqweqfxp
```

Comment:
export TOOLS_ACCURACY_HASH=QmebjcPizAdVFSUAfMBgAGFJhLPVBMvV68LxhSq4LPvv9d
```suggestion
export TOOLS_ACCURACY_HASH=QmYna61uWDGELf4kTEKSfjQaKDUEv9ALSTce6CGqweqfxp
```